script/openqa-bootstrap

```diff
@@ -102,6 +102,18 @@ key = ${API_KEY}
 secret = ${API_SECRET}
 EOF
 
+# configure worker@1 to use x86_64 backends
+cat >> /etc/openqa/workers.ini <<EOF
+[1]
+#WORKER_CLASS = qemu_x86_64_staging,qemu_x86_64
+EOF
+
```

Comment:
again, unfortunately unrelated change. And also the line is commented. I guess you just did that as an experiment to try out?

Reply:
i removed the # in the beginning. i dont think this is unrelated. The job never runs if this is not set

Reply:
qemu_x86_64 should be the default. We have Tumbleweed tests to cover this. Can you show which job fails?

Reply:
No really, because i work on a virtual machine to avoid to touch my openqa. the workers.ini have commented out the workers and the jobs are scheduled but never run if i dont set up the file as such. i assume that you mean that the job picks x86_64 even when the workers are commented somehow.

Reply:
yes